src/test/java/mandel/gameoflife/GridTest.java

```diff
@@ -23,4 +25,48 @@ public void nextGen()
         //then
         assertEquals("0100\n0110\n1010\n0000\n", grid.toString());
     }
+
+    @Test
+    public void loadFromRle() {
+        // given
+        String rleContent = "x = 3, y = 3\nbob$2bo$3o!";
+
+        // when
+        rle.loadFromRle(rleContent);
+        int[][] grid = rle.getGrid();
+
+        //then
+        assertEquals(1, grid[0][1]);
+        assertEquals(1, grid[1][2]);
+        assertEquals(1, grid[2][0]);
+        assertEquals(1, grid[2][1]);
+        assertEquals(1, grid[2][2]);
+
+        assertEquals(0, grid[0][0]);
+        assertEquals(0, grid[0][2]);
+        assertEquals(0, grid[1][1]);
+        assertEquals(0, grid[3][3]);
+    }
+
+    @Test
+    public void decodeRle() {
```

Comment:
Technically this is getting tested twice since it is tested by the previous test